.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.9
+  python: python3
```

Comment:
Allow pre-commit to run with any Python version. (I'm using Python 3.10 locally, because that's what power_perceiver requires)